src/middleware/customTokenAuthentication.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+import logger from 'winston'
+
+import * as client from '../model/clients'
+
+const TOKEN_PATTERN = /^ *(?:[Cc][Uu][Ss][Tt][Oo][Mm]) +([A-Za-z0-9\-._~+/]+=*) *$/
```

Comment:
I think we should move this constant to the constants module

Reply:
Why

Reply:
I agree with you but still, why?

Reply:
Code modularity. We have a module where we put constants